### PR TITLE
rados tier-3/test-rados-pool-functionalities: added shell script

### PR DESCRIPTION
**Pull request by pdhiran**
Added shell scripts to test pool functionalities

Signed-off-by: Pawan Dhiran <pdhiran@redhat.com>

### CODE REVIEW

**Comment by pdhiran** [comment]
http://magna002.ceph.redhat.com/ceph-qe-logs/pawan/cephci-run-6GNQ1K/

File: suites/pacific/rados/tier-2_rados_stretch_mode_upgrade.yaml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 # Suite contains tests related to election strategy and Stretched mode
```
Comment (tintumathew10) [inline suites/pacific/rados/tier-2_rados_stretch_mode_upgrade.yaml:1]: Can we rename this file to follow guidelines? 
tier-2_rados_test-strech-mode-upgrade.yaml

Comment (pdhiran) [reply suites/pacific/rados/tier-2_rados_stretch_mode_upgrade.yaml:1]: Updated the file name according to guidedlines.

File: suites/pacific/rados/tier-2_rados_test-stretch-mode-upgrade.yaml [hunk 2]
```diff
@@ -189,38 +194,38 @@ tests:
...
-
-  - test:
-      name: rgw sanity tests
-      module: sanity_rgw.py
-      config:
-          script-name: test_multitenant_user_access.py
-          config-file-name: test_multitenant_access.yaml
-          timeout: 300
-      desc: Perform rgw tests
+#  - test:
```
Comment (SrinivasaBharath) [inline suites/pacific/rados/tier-2_rados_test-stretch-mode-upgrade.yaml:197]: Please remove the comments. As per polarion this test case is automated.

**Review by SrinivasaBharath — CHANGES_REQUESTED** [review]

File: suites/pacific/rados/tier-2_rados_test-stretch-mode-upgrade.yaml (hunk 2 above)
Comment (pdhiran) [reply suites/pacific/rados/tier-2_rados_test-stretch-mode-upgrade.yaml:197]: Commented the following snippet, as once fi lands into cdn, we need to run these tests again. Rationale added at the top of suite file. The same upgrade test is run with deployment suite too.

Comment (pdhiran) [reply suites/pacific/rados/tier-2_rados_test-stretch-mode-upgrade.yaml:197]: Commented as we can un-comment once the fix lands into CDN for the bug. Complete rationale added at the top of suite

**Review by tintumathew10 — APPROVED** [review]
Looks good

**Review by SrinivasaBharath — APPROVED** [review]

**Pull request merged**